pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -500,6 +500,7 @@ def _populate_accordion(self):
         log.debug("ui: new_devices=%s", [d.name for d in new_devices])
 
         ui_roots = self._storage_playground.roots[:]
+        page_efi = Page(_("UEFI partitions"))
```

Comment:
Perhaps this could be `UnknownPage`, as we don't need the SYSTEM/DATA sorting stuff?